<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>157</width>
    <height>48</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FF7tk Widget Demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_14">
    <item>
     <widget class="QComboBox" name="combo_widget">
      <item>
       <property name="text">
        <string>Select Widget...</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OptionsWidget</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Materia Editor</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ItemListView</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CharEditor</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MetaData Creator</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SlotSelect</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PhsListWidget</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MenuListWidget</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lgp Utility</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LocationViewer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ChocoboManager</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AchievementEditor</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>HexLineEdit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Orientation Widget</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Encounter Table Widget</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="encounterTableGroup">
      <property name="title">
       <string>Encounter Table Widget</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_18">
       <item>
        <widget class="QFrame" name="encounterTableFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="materia_editor_box">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="materiaEditor_controls" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QWidget" name="mestar" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>setStarSize</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="sb_materia_editor_setStarSize">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximum">
                <number>128</number>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QCheckBox" name="cbEditableMateriaCombos">
               <property name="text">
                <string>Editable Combo</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cb_materiaEditor_showPlaceHolderMateria">
               <property name="text">
                <string>Show PlaceHolder Materia</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cb_materia_editor_setEditable">
               <property name="text">
                <string>is Editable</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="editor_box">
         <property name="title">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="char_editor_box">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Initialize Char Data</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="cb_charEditor">
              <property name="text">
               <string>is Editable</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>Debug</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_3">
            <property name="text">
             <string>Editable Combos </string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="charEditor_box">
         <property name="title">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="phsListBox">
      <property name="title">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="menuListBox">
      <property name="title">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="dialog_preview_box">
      <property name="title">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="metadata_box">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btn_showmetaData">
         <property name="text">
          <string>Show Dialog</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="slotSelect_Box">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="cbShowLoad">
         <property name="text">
          <string>Allow Loading of new file from dialog</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_slotSelect">
         <property name="text">
          <string>Select File to Preview</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="lgp_Box">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>View / Extract</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QPushButton" name="btn_lgpSelect">
            <property name="text">
             <string>Select Lgp To Preview</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupLgpView">
            <property name="title">
             <string/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="lblLgpCompany">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblLgpProduct">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="listLgpFile"/>
             </item>
             <item>
              <widget class="QPushButton" name="btnExtractLgp">
               <property name="text">
                <string>Extract To...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Create / Add File(s)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Demo Coming Soon</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="locListBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>Advanced Mode</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lbl_region">
            <property name="text">
             <string>Region</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboRegion">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
            <item>
             <property name="text">
              <string>FF7-S01BASCUS-94163</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FF7-S01BESCES-00867</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FF7-S01BESCES-00868</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FF7-S01BESCES-00869</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FF7-S01BESCES-00900</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FF7-S01BISLPS-00700</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FF7-S01BISLPS-01057</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="innerLocBox">
         <property name="title">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="ChocoboManagerBox"/>
    </item>
    <item>
     <widget class="QGroupBox" name="AchievementEditor_Box">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QPushButton" name="btn_loadAchievement">
         <property name="text">
          <string>Select Achievement to Preview</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="achievementEditor_Frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_saveAchievement">
         <property name="text">
          <string>Save File</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="itemListView_Group" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>53</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <widget class="QWidget" name="listView_Controls" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_17">
          <item>
           <widget class="QCheckBox" name="cb_itemSelection_showPlaceHolders">
            <property name="text">
             <string>Show PlaceHolder Items</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_itemSelectionDeleageEditable">
            <property name="text">
             <string>Editable Combos</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Maximum Quantity</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sb_itemListViewMaxQty">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximum">
               <number>127</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="itemListView_Box">
         <property name="title">
          <string>ItemListView</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="hexLineEdit_group">
      <property name="title">
       <string>HexLineEdit</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>MaxLength</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="sb_hexEditLine_maxlen">
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="maximum">
            <double>20000000000000000426408380189087937446025157425359298935486676992.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="hexLineEdit_Box">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="orientationGroup">
      <property name="title">
       <string>Orientation Widget</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_16">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Style</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_orientationWidget_style">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Grid</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Arrow</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Zero Direction</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_orientationWidget_zeroDirection">
           <property name="currentIndex">
            <number>3</number>
           </property>
           <item>
            <property name="text">
             <string>North</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>East</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>South</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>West</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Rotation Direction</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_orientationWidget_rotationDirection">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <item>
            <property name="text">
             <string>Clockwise</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Counter Clockwise</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_orientationWidget_readOnly">
         <property name="text">
          <string>Read Only</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb_orientationWidget_value">
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string/>
         </property>
         <property name="prefix">
          <string>Value: </string>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="orientationFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
